### DISCUSSION TITLE
can i do anything with this ext?

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1282] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
chrome://extensions/?id=jjckigopagkhaikodedjnmbccfpnmiea

**Comment by ghost** [comment, marked as answer]
probably Not you should check for unsafe eval, mby using Ctrl + f on the manifest page , if there is which I doubt there is , you would be able to use rigtools with it

**Reply by Sam-hacks1** [reply]
how do I get on the manifest page

**Reply by ghost** [reply]
here's the link : chrome-extension://jjckigopagkhaikodedjnmbccfpnmiea/manifest.json

**Reply by ghost** [reply]
do control + f , and type in unsafe and it should see if unsafe is in it

**Reply by Sam-hacks1** [reply]
yea it isn't in it but thanks
